src/helpers/umzug-helper.js

```diff
@@ -15,6 +15,8 @@ const storageJsonName = {
   seeder: 'sequelize-data.json'
 };
 
+let timestampsDefault = false;
```

Comment:
I don't like using this out of scope variable to track timestamp default

Reply:
Open to suggestion - how else would you want to track this? I wanted to keep this a limited module variable - but it can be modified to be as a variable in the `module.exports`

Reply:
@sushantdhiman ?

Reply:
I don't see any good way either, we can keep this